userdocs/src/usage/managing-nodegroups.md

```diff
@@ -160,6 +160,35 @@ There are no specific commands in `eksctl`to update the labels of a nodegroup bu
 kubectl label nodes -l alpha.eksctl.io/nodegroup-name=ng-1 new-label=foo
 ```
 
+### SSH Access
```

Comment:
We don't really have a great place for these docs to live, and I don't really want to add a page for just SSH configuration. I've put it here for now, open to suggests